scene/2d/animated_sprite.cpp

```diff
@@ -490,6 +490,34 @@ Point2 AnimatedSprite::get_offset() const {
 	return offset;
 }
 
+void AnimatedSprite::set_anchor(const Point2 &p_anchor) {
+
+	anchor = p_anchor;
+	anchor.x = CLAMP(anchor.x, -1, 1);
+	anchor.y = CLAMP(anchor.y, -1, 1);
```

Comment:
I'll remove the clamp. Out of curiosity, what would be a use case for an unclamped anchor?
(I couldn't think of any, which is why I had the value clamped)